### DISCUSSION TITLE
Can I split the context sections into different tty window?

### DISCUSSION THREAD

**Discussion by zhouguoqionghai** [hugsy/gef#765] [Q&A]
As an example, can I show "legend regs stack code source" in a tty window and "memory" in another?

**Comment by hugsy** [comment]
If you use `screen` or `tmux`, simply use `tmux-setup` it'll do it automatically for you (see https://gef.readthedocs.io/en/master/commands/tmux-setup/)

If not, you'll need to use the config option `redirect` to point to the file/tty you want to drop your context to (see https://gef.readthedocs.io/en/master/commands/context/#redirecting-context-output-to-another-ttyfile)

**Reply by zhouguoqionghai** [reply]
Thanks for your reply @hugsy.  I already tried `redirect` a few days before, it drops all context sections to another tty windows. I wander if there is a feature to redirect some sections but not all of them to another tty window?

**Reply by hugsy** [reply]
GEF doesn't support that natively, but many people I know use [voltron](https://github.com/snare/voltron) for that. 
Have you tried it ? See https://gef.readthedocs.io/en/master/faq/#does-gef-prevent-the-use-of-other-gdb-plugins

**Reply by hugsy** [reply]
I also noticed we had an open ticket for it (https://github.com/hugsy/gef/issues/554) but since it seems a minor feature that can be achieved by other means (here `voltron`), the feature has become stale. If you feel like it, you can PR it.
